dist/src/main/java/io/camunda/application/StandaloneSchemaManager.java

```
@@ -100,15 +111,73 @@ public static void main(final String[] args) throws IOException {
 
   @Override
   public void run(final String... args) throws Exception {
-    if (brokerProperties.getExporters().containsKey("elasticsearch")) {
-      final var elasticsearchConfig =
-          new ExporterConfiguration(
-                  "elasticsearch", brokerProperties.getExporters().get("elasticsearch").getArgs())
-              .instantiate(ElasticsearchExporterConfiguration.class);
-      new ElasticsearchExporterSchemaManager(elasticsearchConfig)
-          .createSchema(CURRENT_BROKER_VERSION.toString());
+    try {
+      LOG.info("Creating/updating schema for Camunda using schema manager");
+      getSchemaCreators()
+          .forEach(
+              schemaCreator -> {
+                schemaCreator.createSchema(CURRENT_BROKER_VERSION.toString());
+              });
+      LOG.info("... finished creating/updating schema for Camunda");
+    } catch (final Exception e) {
+      LOG.error("Failed to create/update schema", e);
     }
```

🔧 we should re-throw the exception (or remove the catch exception), so that the application exits with non zero value